File: fix_zIndex.js
const fs = require('fs');
let code = fs.readFileSync('src/components/map/LeafletGisMap.tsx', 'utf8');

// The right panel needs to be visible over the map
code = code.replace(
  /absolute top-4 left-0 bottom-6 z-\[1001\]/,
  "absolute top-4 left-0 bottom-6 z-[1001]"
);

code = code.replace(
  /absolute top-4 right-0 bottom-6 z-\[1001\]/,
  "absolute top-4 right-0 bottom-6 z-[1001]"
);

fs.writeFileSync('src/components/map/LeafletGisMap.tsx', code);

File: fix_zIndex.py
import re

with open('src/components/map/LeafletGisMap.tsx', 'r') as f:
    content = f.read()

# Make sure map has z-0 
content = content.replace('id="leaflet-gis-map-canvas"', 'id="leaflet-gis-map-canvas"')
content = content.replace('z-[1001]', 'z-[1001]')

with open('src/components/map/LeafletGisMap.tsx', 'w') as f:
    f.write(content)
